File: data/sample.rb
# The Hello class
class HelloWorld
  Version = "1.0"

  def initialize(myname="Ruby")
    @name = myname
  end

  def hello
    print "Hello, world. I am ", @name, ".\n"
  end
end

# Create new objects
bob   = HelloWorld.new("Bob")
alice = HelloWorld.new("Alice")
ruby  = HelloWorld.new

# Output
bob.hello
alice.hello
ruby.hello

File: data/sample.py
class HelloWorld(object):
    def __init__(self, message):
        self.message = message

    def hello(self):
        print "Hello, world. I am {}".format(self.message)

if __name__ == "__main__":
    bob = HelloWorld('Bob')
    alice = HelloWorld('Alice')
    bob.hello()
    alice.hello()
